contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,571 @@
+# Local Raw Block Consumption via Persistent Volume Source
```

Comment:
I think it's implying that we're not going to support direct volume access (ie VolumeSource in the pod)